.github/workflows/unstale.yml

```diff
@@ -0,0 +1,19 @@
+name: Unmark issues and pull requests as stale on activity
+on:
+  issue_comment:
+    types: [created]
+
+# actions/stale does this automatically, but only once a day.
+# This immediately removes the label when the user creates a comment.
+jobs:
+  remove-stale-label:
+    if: (github.repository == 'rubocop/rubocop')
```

Comment:
I believe this workflow only runs in `rubocop/rubocop`. Is this condition necessary?

Reply:
`stale.yml` has the same condition, and I think it's good practise to exclude such project-specific things even if issues/comments and such on forks are rather unlikely.

Reply:
It is necessary or else it runs on forks pointlessly (and sends emails too). It was limited for the stale action in #13730.

Reply:
It's not so important fot this one because it only runs when a comment is created, the main action is on a schedule

Reply:
Ah, that's right. I overlooked that.